packages/botonic-plugin-flow-builder/src/api.ts

```diff
@@ -244,13 +244,12 @@ export class FlowBuilderApi {
     return this.flow.is_knowledge_base_active || false
   }
 
-  getResolvedLocale(locale: string): string {
-    if (this.flow.locales.find(flowLocale => flowLocale === locale)) {
+  getResolvedLocale(locale?: string): string {
+    if (locale && this.flow.locales.find(flowLocale => flowLocale === locale)) {
```

Comment:
not sure if `locale &&` is needed as you will already match with find if flowLocale is equal to locale. we can try it, ow, it's okay as is